example-workflows/Technique_2_Slack_API_Method/README.md

```diff
@@ -0,0 +1,40 @@
+# Technique 2: Slack API method
+
+A bot token or user token or [token of some other kind][tokens] must be used to
+call one of [the Slack API methods][methods] with this technique.
+
+## Setup
+
+For details on how to setup this technique in GitHub Actions, read the
```

Comment:
```suggestion
For details on how to set up this technique in GitHub Actions, read the
```

Reply:
I refuse to believe a rules exists for phrasewords like "setup" and "login". Or the rule is "do the opposite of what seemed right" 😉